src/NuGet.Clients/NuGet.PackageManagement.UI/Resources.resx

```diff
@@ -844,4 +844,22 @@ Please see https://aka.ms/troubleshoot_nuget_cache for more help.</value>
   <data name="Accessibility_PackageDetailsUrlLink" xml:space="preserve">
     <value>Package details link</value>
   </data>
+  <data name="Label_Deprecated" xml:space="preserve">
+    <value>Deprecated</value>
+  </data>
+  <data name="Label_OrAbove" xml:space="preserve">
+    <value> (or above)</value>
```

Comment:
or higher maybe?

Reply:
The [spec](https://github.com/NuGet/Home/wiki/Deprecate-packages#visual-studio) says ` (or above)`. What's your take on this @anangaur?

Reply:
Maybe it's cultural, but I would vote for higher. To me, the word above relates to physical space, although online dictionaries suggest this usage is also in use.

Reply:
Changed it to ` (or higher)` in https://github.com/NuGet/NuGet.Client/pull/2899/commits/0c1697551522569a53f7eca9790cf2b4d237dbf5